autosklearn/evaluation/__init__.py

```diff
@@ -85,11 +98,17 @@ def fit_predict_try_except_decorator(
         queue.close()
 
 
-def get_cost_of_crash(metric: Scorer) -> float:
+def get_cost_of_crash(
+    metric: Union[Scorer | Sequence[Scorer]],
+) -> Union[float, List[float]]:
 
-    # The metric must always be defined to extract optimum/worst
-    if not isinstance(metric, Scorer):
-        raise ValueError("The metric must be stricly be an instance of Scorer")
+    if isinstance(metric, Sequence):
+        return [cast(float, get_cost_of_crash(metric_)) for metric_ in metric]
+    elif not isinstance(metric, Scorer):
```

Comment:
is that even possible?

Reply:
I don't know, I left it there because we had this check in place before.